index.js

```diff
@@ -1,25 +1,35 @@
-var _ = require('lodash');
-var loaderUtils = require('loader-utils');
+import _ from 'lodash'
+import loaderUtils from 'loader-utils'
 
-function getOptions(context) {
+class UnsupportedConfiguration extends Error {}
+
+function getOptions (context) {
   if (context.options && context.options.ejsLoader) {
-    return context.options.ejsLoader;
+    return context.options.ejsLoader
```

Comment:
Why the change to drop semicolons?

Reply:
@jessewagstaff I dropped them from the linter because we are now using babel to transform the file. They aren't required by default and are added in the bundle.